inc/knowbaseitem.class.php

```diff
@@ -412,15 +412,15 @@ static function addVisibilityJoins($forceall = false) {
       Toolbox::deprecated('Use getVisibilityCriteria');
 
       //get and clean criteria
-      $criteria = self::getVisibilityCriteria();
+      $criteria = self::getVisibilityCriteria(true);
```

Comment:
Are you sure true is ok in all cases ? Maybe using `$forceall` value is more relevant.
```suggestion
      $criteria = self::getVisibilityCriteria($forceall);
```